### PR TITLE
6 getuser

### PR DESCRIPTION
**Pull request by zaghloulegy**
solved get array of  username 
create 2 files user model and user controller and build a test

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -156,5 +156,22 @@ describe("patchReviewVotes() PATCH /api/reviews/:review_id", () => {
   });
 });
 
+describe.only("getUsers GET /api/users", () => {
```
Comment (dappernerddesigns) [inline __tests__/app.test.js:159]: Please remove this .only. All tests must be passing before a commit and PR

**Review by dappernerddesigns — CHANGES_REQUESTED** [review]
Please remove the .only in your test file and re submit your pull request. All tests must be passing.

File: __tests__/app.test.js (hunk 1 above)
Comment (zaghloulegy) [reply __tests__/app.test.js:159]: i didn`t know that

Comment (zaghloulegy) [reply __tests__/app.test.js]: done and it`s all passed

**Pull request merged**